model/buildlogger_test.go

```diff
@@ -378,7 +378,7 @@ func TestBuildloggerAppend(t *testing.T) {
 		}
 		var filenames map[string]bool
 		var actualData []byte
-		for i := 0; i < 10; i++ {
+		for i := 0; i < 20; i++ {
```

Comment:
This is annoying, but even with the 10 retries it fails quite often on macs and once in a while on linux.

Reply:
Do you know why it's slow? As far as I see, Append is synchronous so I would imagine it would be written to the filesystem without needing to periodically retry.